cmd/bootmap.c

```diff
@@ -0,0 +1,50 @@
+#include <common.h>
+#include <command.h>
+#include <console.h>
+
+static int do_get_active_bootmap(cmd_tbl_t * cmdtp, int flag, int argc, char * const argv[])
+{
+	void *addr;
+	struct bootmap_switch {
+		u32 address;
+		u32 size;
+	} *bms;
+
+	if (argc < 2)
+		return CMD_RET_USAGE;
+
+	addr = (void *)simple_strtoul(argv[1], NULL, 16);
+	if (!addr) {
+		printf("Bootmap-Switch address is NULL!\n");
+		return CMD_RET_FAILURE;
```

Comment:
Perhaps it's better to use `CMD_RET_USAGE` here and below instead?

Reply:
As this error isn't due to a misuse of the command, I think it's better to through an error. Otherwise, a user might think she provided the wrong arguments.

Reply:
I believe it can only return `NULL` if either the command is incorrectly called, or user gives a number which is not a valid hex pointer as the argument. Both cases are considered as misuse, and in this case it is beneficial to show the help (if it is enabled via _SYS_LONGHELP_).